packages/node-core/src/indexer/blockDispatcher/worker-block-dispatcher.ts

```diff
@@ -115,7 +113,7 @@ export abstract class WorkerBlockDispatcher<DS, W extends Worker, B>
       let startIndex = 0;
       while (startIndex < heights.length) {
```

Comment:
You can remove this loop.

Reply:
Why? This code doesn't relate to the PR or changes

Reply:
Because after the modification, it now only executes the loop once.